BDArmory/WeaponMounts/ModuleTurret.cs

```diff
@@ -369,9 +369,32 @@ void SetupStandbyLocalRotation(BaseField field = null, object obj = null)
             yawStandbyAngleEd.maxValue = yawRange / 2f;
             yawStandbyAngle = Mathf.Clamp(yawStandbyAngle, yawStandbyAngleEd.minValue, yawStandbyAngleEd.maxValue);
             yawStandbyAngleEd.onFieldChanged = OnStandbyAngleChanged;
-            OnStandbyAngleChanged();
+            SetStandbyAngle();
         }
         void OnStandbyAngleChanged(BaseField field = null, object obj = null)
+        {
+            if (standbyLocalRotation != null)
```

Comment:
The `null` check is unnecessary since you're calling `SetStandbyAngle();` in `SetupStandbyLocalRotation` which will make it non-`null` on start-up.